gnome-initial-setup/pages/branding-welcome/gis-branding-welcome-page.c

```diff
@@ -214,6 +226,12 @@ gis_branding_welcome_page_constructed (GObject *object)
   update_line_height_for_description (page);
   load_css_overrides (page);
 
+  /* Use ctrl+f to show factory dialog */
+  priv->accel_group = gtk_accel_group_new ();
+  closure = g_cclosure_new_swap (G_CALLBACK (gis_page_show_factory_dialog), page, NULL);
+  gtk_accel_group_connect (priv->accel_group, GDK_KEY_f, GDK_CONTROL_MASK, 0, closure);
+  g_closure_unref (closure);
```

Comment:
I'm just following the lead of the language page here, and I'd rather leave it like this for now if it's ok, to avoid even more refactoring.